2025-10-15 14:39:11,399 [processing](INFO): Initializing backseat process (pilot lrauv based on marl)
2025-10-15 14:39:14,314 [Logger](CRITICAL): Uncaught exception
Traceback (most recent call last):
  File "./backseat_app/main.py", line 111, in <module>
    main()
  File "./backseat_app/main.py", line 107, in main
    backseat_app.spin()
  File "./backseat_app/main.py", line 75, in spin
    self.marl_processor.compute_new_heading()
  File "/home/maiv/lrauv_sim/lrauv-backseat-marl/backseat_app/processing.py", line 163, in compute_new_heading
    self.new_action = self.rl_tracking.newAction(self.target_address,-1,self.lrauv_pose,self.lrauv_depth,self.target_timestamp,new_range=False)
  File "/home/maiv/lrauv_sim/lrauv-backseat-marl/backseat_app/utils.py", line 940, in newAction
    print('INFO: LRAUV pos (x,y,depth,yaw)= %.2fm, %.2fm, %.2fm, %.2fdegrees'%(self.lrauv_position[0],self.lrauv_position[2],lrauvDepth,angle*180./np.pi))
NameError: name 'angle' is not defined
